### PR TITLE
(Feature) Expose Address and Etherscan link / Show Token Tooltip

### LINKED ISSUE #352: [FE] We should come up with some reasonable naming convention for wrapped outcome tokens
Right now, when you wrap an outcome token, it is labelled "WMT".

image

What does that mean?

We should come up with a naming convention for these tokens.

Something like `Wrapped [DAI C:0xba11...501d O:0]` probably makes sense.

### LINKED ISSUE #353: [UX/UI] Expose address and etherscan link for wrapped tokens
Once you wrap a token, there is no easy way to figure out that token's address.

We should provide a way to copy the address along with a link to the token on Etherscan.

image

### PR DESCRIPTION
**Pull request by gabitoesmiapodo**
Closes #353
Closes #352

I'm not completely sure this is 100% correct, it seems TheGraph is not indexing the latest transactions since yesterday, but here it goes anyways...

- Changed "Collateral Contract" to "Collateral Address" to match the search options' items. "Collateral Contract Address" would be more adequate, but it won't fit into the details nor the search options' dropdown.
- Added Etherscan's links to open in the same or a new tab to "Collateral Address".
- Added tooltip to "WMT" (wrapped collateral symbol) as suggested [here](https://github.com/gnosis/conditional-tokens-explorer/issues/352#issuecomment-698578826) and [here](https://github.com/gnosis/conditional-tokens-explorer/issues/352#issuecomment-700951981). There's not much else we can do for now, until a better naming convention is implemented.
- Added "Wrapped Collateral Address" to the details. It allows to copy the address, and it's got links to open Etherscan in the same or a new tab as requested on https://github.com/gnosis/conditional-tokens-explorer/issues/353
- As stated, there's something going on with TheGraph, but you can check the wrap / unwrap transactions in the "Wrapped Collateral Address" Etherscan link ("Transfers" tab). Transactions work, TheGraph doesn't.

Screen Shot 2020-10-06 at 22 38 42

Screen Shot 2020-10-06 at 22 53 54

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr437--conditionaltokensexplorer.review.gnosisdev.com

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr437--conditionaltokensexplorer.review.gnosisdev.com

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr437--conditionaltokensexplorer.review.gnosisdev.com

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr437--conditionaltokensexplorer.review.gnosisdev.com

File: src/util/tools.ts [hunk 1]
```diff
@@ -379,3 +380,16 @@ export const isPartitionFullIndexSet = (
 
 export const capitalize = (str: string): string =>
   str.charAt(0).toUpperCase() + str.slice(1).toLowerCase()
+
+export const getNetworkName = (networdId: number) => {
```
Comment (mariano-aguero) [inline src/util/tools.ts:384]: This should be part of the networkConfig class, I will fix it

**Review by mariano-aguero — APPROVED** [review]

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr437--conditionaltokensexplorer.review.gnosisdev.com

**Pull request merged**